core/db/migrate/20141101231208_fix_adjustment_order_presence.rb

```diff
@@ -5,7 +5,7 @@ def change
       adjustable = adjustment.adjustable
       if adjustable.is_a? Spree::Order
         adjustment.update_attributes!(order_id: adjustable.id)
-      else
+      elsif adjustable && adjustable.order
         adjustment.update_attributes!(adjustable: adjustable.order)
```

Comment:
I'm thinking you're right that it should be:
`adjustment.update_attributes!(order: adjustable.order)`

The commit introducing it even has that as the last bullet point:
https://github.com/brchristian/spree/commit/87cc12debce9d6a54a709281380bc3d4bb607973

With the commit accepted it had typos in the migration and was followed up with change to:
https://github.com/brchristian/spree/commit/d6c18fcd5774b624bef4e0684af4d7910b9dff1a